_docs/addons/yform/yf_datatable_XofY.md

```diff
@@ -43,22 +43,21 @@ YForm-EP vor der Tabellenausgabe angezogen wird.
 
 ```php
 \rex_extension::register('YFORM_DATA_LIST',
-    function( \rex_extension_point $ep )
-    {
+    static function (rex_extension_point $ep) {
```

Comment:
Ich hab damals bewußt `\rex_extension_point` geschrieben, weil es dann einfacher ist, den Code in ein bestehendes Umfeld mit Namespaces zu kopieren. Ohne Namespaces stört es nicht und dr Fixer bereinigt es dann eh. Was umgekehrt nicht gilt.

Reply:
Ich habe das in einem Addon ohne Namespace.